web-app/js/cookies-helper.js

```diff
@@ -48,7 +48,6 @@ var cookiesHelper = {
         if (cookiesInfo) {
```

Comment:
This var has the same name as the object with internal functions. Is it working?